lambdas/indexer/.chalice/config.json.template.py

```diff
@@ -20,20 +20,17 @@
     "stages": {
         config.deployment_stage: {
             "lambda_functions": {
-                # FIXME: Brittle coupling between the string literal below and
-                #        the handler function name in app.py
-                #        https://github.com/DataBiosphere/azul/issues/1848
-                "contribute": {
+                config.contribute_lambda_basename: {
                     "reserved_concurrency": config.indexer_concurrency,
                     "lambda_memory_size": 256,
                     "lambda_timeout": config.contribution_lambda_timeout,
                 },
-                "aggregate": {
+                config.aggregate_lambda_basename: {
```

Comment:
See if you can import the app module here and extract the function name from the handler like so

```suggestion
                config.lambda_basename(app.aggregate): {
```

Reply:
I don't understand why this is better.